src/framework/components/element/text-element.js

```diff
@@ -28,7 +28,12 @@ Object.assign(pc, function () {
         this._text = "";
         this._i18nKey = null;
 
-        this._fontAsset = null;
+        this._fontAsset = new pc.LocalizedAsset(this._system.app);
+        this._fontAsset.disableLocalization = true;
+        this._fontAsset.on('load', this._onFontLoad, this);
```

Comment:
I don't see where we fire these events for the default asset?

Reply:
These events are fired on the pc.LocalizedAsset instance